core/transactor/transactor.go

```diff
@@ -42,9 +41,9 @@ type Transactor struct {
 }
 
 // NewTransactor creates and returns new Transactor instance
-func NewTransactor(bindAddress, endpointAddress, registryAddress, accountantID, channelImplementation string, signerFactory identity.SignerFactory) *Transactor {
+func NewTransactor(httpClient *requests.HTTPClient, endpointAddress, registryAddress, accountantID, channelImplementation string, signerFactory identity.SignerFactory) *Transactor {
 	return &Transactor{
-		http:                  requests.NewHTTPClient(bindAddress, 20*time.Second),
+		httpClient:            httpClient,
```

Comment:
In [some](https://github.com/mysteriumnetwork/node/pull/1438/files#diff-4801e8de631b36d8ef7e26fabad34ae2R83) places you are calling it  `http` in other it's `httpClient`. It should be consistent everywhere.